database/instance/manage-db.sh

```diff
@@ -80,8 +81,8 @@ function createContainer {
 	echo "[ARA] Creating the image..."
 	run 'docker build -t ara-db-image .'
 	echo "[ARA] Create the $CONTAINER_NAME container (data in $dataDir)"
-	docker_cmd=$(forgeCreationCommand)
-	docker_cmd="$docker_cmd -v /$dataDir:/var/lib/mysql" # Starting-slashes are for Windows compatibility
+	docker_cmd=$(forgeCreationCommand $dataDir)
```

Comment:
I think this fix another issue, maybe it should be in another PR ?

Reply:
with what purpose ?